Add documentation for Apache webservers.

Petraea opened the pull request.

nsoranzo commented on sessions/10-uwsgi/ex1-uwsgi.md:
```diff
@@ -118,7 +133,24 @@ Then, save and quit your editor. Restart nginx with:
-**Part 2 - Run Galaxy with uWSGI**
+**Part 2 - Configure Apache for uWSGI**
+
+We previously configured Apache HTTP to expect the main paster process to appear on port 8080, but we want it to communicate with uWSGI on port 4001 instead. Edit the file `/etc/apache2/sites-enabled/000-galaxy.conf` and update the line:
```
We need to be consistent in `sessions/03-production-basics/ex4-apache.md` with the configuration file name, it seems both `000-galaxy.conf` and `galaxy.conf` are used. I'd prefer the latter, which is the same we use for nginx.

Petraea commented:
No problem. I've updated that. I've also found how to get Apache to communicate over the same socket style as nginx using mod-proxy-uwsgi and added the config here.

nsoranzo commented on sessions/10-uwsgi/ex1-uwsgi.md:
````diff
@@ -118,7 +118,31 @@ Then, save and quit your editor. Restart nginx with:
+We previously configured Apache HTTP to expect the main paster process to appear on port 8080, but we want it to communicate with uWSGI on port 4001 vit its native protocol instead. Apache alas doesn't come with this as standard, but the module can be installed with:
+
+```console
+apt-get install libapache2-mod-proxy-uwsgi
````
The [Galaxy documentation](https://docs.galaxyproject.org/en/latest/admin/scaling.html) uses `mod_uwsgi`, but that seems `mod_proxy_uwsgi` is what [uWSGI documentation](https://uwsgi-docs.readthedocs.io/en/latest/Apache.html) recommends. So probably we need to update the Galaxy docs.
